index.html

```diff
@@ -909,7 +909,7 @@
 Services sometimes use people's data in order to protect those or other people.
 A service that does this should explain what data it's
 using for this purpose. It should also say how it might use or share a person's
-data if it believes that person has broken the rules.
+data if it believes that person has violated the service's norms.
```

Comment:
```suggestion
data if it believes that person has violated the service's rules.
```
Adding "the service's" seems useful, but norms looks like a fuzzier bound than rules, providing less certainty to users.

Reply:
That sounds like a good point to me. To maintain the fix to @chrisn's issue, I changed all the uses of "norms" to "rules" in this section. How does that wind up looking?

Reply:
Seems good to me